plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -49,6 +49,9 @@ Play="Play"
 Pause="Pause"
 Stop="Stop"
 
+FFmpeg.Directives="FFmpeg Directives"
+FFmpeg.Directives.Tooltip="Allows you to provide extra parameters to FFmpeg. This only accepts simple parameters in a key=value format, eg rtsp_transport=tcp\nMultiple parameters can be passed provided they are separated by a space, eg rtsp_transport=tcp rtsp_flags=prefer_tcp\n"
```

Comment:
Use the word "options" instead of "parameters".

Reply:
done